assets/components/cert-manager/manifests/templates/namespace.yaml

```diff
@@ -1,9 +0,0 @@
-apiVersion: v1
-kind: Namespace
-metadata:
-  name: {{ .Release.Namespace }}
-  {{- if .Values.webhook.enabled }}
-  labels:
-    certmanager.k8s.io/disable-validation: "true"
```

Comment:
This commit along with the previous one should be part of a separate PR.

While removing release namespace manifests, care should be taken that `labels` and `annotations` are not lost, since `helm.CreateNamespace=true` creates a namespace on the fly before installing the component.

Please add any such labels and annotations for all the components as review comments to this PR  #646

Reply:
this commit is removed so resolving.